reconcile/queries.py

```diff
@@ -21,100 +26,34 @@
 """
 
 
-def get_secret_reader_settings() -> Optional[Mapping[str, Any]]:
+def get_secret_reader_settings() -> Mapping[str, Any]:
```

Comment:
nit: typically the idea is accept general inputs (`Mapping`), but be more specific on outputs (`dict`). This is covered here: https://service.pages.redhat.com/dev-guidelines/docs/appsre/python-style-guide/#type-hints

Not going to block on that change.